### ISSUE TITLE
Add accessibility support through Aria tags

### ISSUE THREAD

**Issue by errant01** [mattlewis92/angular-calendar#941]
**Is your feature request related to a problem? Please describe**
Screen reader (like ChromeVox) cannot successfully guide user understandably through reading and interacting with the month view. I suspect other views have problems also

**Describe the solution you'd like**

Screen Reader can successfully guide a user through all calendar views. In most cases this is done using the set of ARIA tags to augment the template html to give hints for the screen reader and correct any tab-ordering issues.

**Describe your use case for implementing this feature**

Our application needs to be compatible with and support users with screen readers.

**Additional context**

**Comment by mattlewis92** [comment]
Hey this would be something I'd love to support, however I have literally no knowledge about how to do this, feel free to send a PR to add the appropriate ARIA tags! 😄

**Comment by vandents** [comment]
@mattlewis92 my team and I are looking to contribute to the project by adding some better accessibility support, as this is a requirement for our use-case as well as others'. Such a change would solve this issue.

We are wondering if you have any suggestions for best-practices or guidelines for adding features to your project. You may email me or just mention me here if you want.

**Comment by mattlewis92** [comment]
@vandents thanks, that would be really awesome! Initial setup guidelines are here: https://github.com/mattlewis92/angular-calendar#development

In terms of best practices and guidelines, I try and minimise breaking changes as much as possible (I don't think adding aria tags should require any). Also the project follows the [conventional commits spec](https://www.conventionalcommits.org/en/v1.0.0-beta.4/) so that commits are recorded automatically in the changelog.

If you get stick with anything, please just let me know and I'll do what I can to help 😄

**Comment by mattlewis92** [comment]
Thanks to the hard work of @vandents and @JaxonWright, support for this was just added to the next beta. Please give it a try with:

```
npm i angular-calendar@next
```

Note this should not be a breaking change for your apps 🎉

**Comment by JaxonWright** [comment]
Yay. It's finally in stable!
